README.md

```diff
@@ -138,3 +138,109 @@ In a new terminal you can open the Gazebo GUI to see the environment and the rob
 
 
 Click on the menu bar "File -> Open Config", navigate to "~/indigo/src/mas_industrial_robotics" and select the "youbot.rviz" file.
+
+## Errors/problems encountered during build and their possible solutions
+### Unable to access 'unmerged_packages_for_testing' repo
+This is a private repository. If you do not have access to this repo, running ''./repository.debs'' will return an error.
+
+_Solution_: Request access from Deebul Nair or Santosh Thoduka.
+
+### Package 'costmap_2d' takes too long to build
+This is experienced on VM setups.
+
+_Solution_: Try building only this package. Generally takes around 16-20 minutes depending on PC configuration.
+
+     catkin build costmap_2d
+
+### Error related to 'KDL'
+When building package ''mcr_arm_cartesian_control'', following error could be encountered:
+
+     int error_sigma = ((KDL::ChainIkSolverVel_wdls*) ik_solver)->getSigma(sigma);
+
+_Solution_: As suggested in this [thread], comment the line in the source file located in folder ''mcr_manipulation/mcr_arm_cartesian_control/common/src/arm_cartesian_control.cpp''.
+
+[thread]: https://github.com/b-it-bots/mas_common_robotics/commit/00ce574505b9571fd0299e0684790341c1b8fdf4
+
+### Errors related to uninstalled packages
+There are certain packages needed as prerequisite. 
+
+_Solution_: In case you do not have those, please google the relevant package, clone the git repo to your local machine, then build it. Following packages are generally found to be missing.
+
+#### [brics_actuator]
+[brics_actuator]: http://wiki.ros.org/brics_actuator
+Clone the git repo to your machine:
+
+     git clone git@github.com:wnowak/brics_actuator.git
```

Comment:
There is a `ros-kinetic-brics-actuator` package in `apt`. Why shouldn't one just install using `sudo apt install ros-kinetic-brics-actuator` instead?

Reply:
@RaghuvirShirodkar  ?